web/ui-customization/README.md

```diff
@@ -0,0 +1,6 @@
+# UI Customization example
+
+This is an example for building a custom comment thread UI using the new [customization slots API](https://www.nutrient.io/guides/web/user-interface/ui-customization/introduction/).
+It replicates default functionalities of creating, editing and deleting comments.
+
+We use React and [Baseline UI](https://www.nutrient.io/baseline-ui) to make it simpler to build the UI.
```

Comment:
But how does it work? `pnpm install`, etc?

Reply:
Added in https://github.com/PSPDFKit/awesome-nutrient/pull/151/commits/d1aed8d7fec665f8e0a4b9d00db27812b402b8c3.